lib/desired-caps.js

```diff
@@ -0,0 +1,7 @@
+let geckodriverConstraints = {
+  geckodriverPort: {
+    isNumber: true
+  }
```

Comment:
I would also add a capability to manually provide the path to geckodriver

Reply:
Added!